clarify auth0 uri (#8682)
fishtown-analytics/docs.getdbt.com: website/snippets/access_url.md

> The following steps use `YOUR_AUTH0_URI` and `YOUR_AUTH0_ENTITYID`, which need to be replaced with the [appropriate Auth0 URI (Callback URI) and Auth0 Entity ID](/docs/cloud/manage-access/sso-overview#auth0-uris) for your region. You can find these values in **Account settings** > **SSO & SCIM** > **Edit** after selecting your identity provider.

Author: ```suggestion
The following steps use `YOUR_AUTH0_URI` and `YOUR_AUTH0_ENTITYID`, which need to be replaced with the [appropriate Auth0 URI and Auth0 Entity ID](/docs/cloud/manage-access/sso-overview#auth0-uris) for your region. You can find these values in **Account settings** > **SSO & SCIM** > **Edit** or **Get started** after selecting your identity provider.
```

> The following steps use `YOUR_AUTH0_URI` and `YOUR_AUTH0_ENTITYID`, which need to be replaced with the [appropriate Auth0 URI and Auth0 Entity ID](/docs/cloud/manage-access/sso-overview#auth0-uris) for your region. You can find these values in **Account settings** > **SSO & SCIM** > **Edit** or **Get started** after selecting your identity provider.

Reviewer: ```suggestion
The following steps use `YOUR_AUTH0_URI` and `YOUR_AUTH0_ENTITYID`. Replace these placeholders with the [appropriate Auth0 URI and Auth0 Entity ID](/docs/cloud/manage-access/sso-overview#auth0-uris) for your region. You can find these values in **Account settings** > **SSO & SCIM** > **Edit** or **Get started** after selecting your identity provider.
```
